bin/README.md

```diff
@@ -31,7 +31,7 @@ For a more permanent solution you might have in your `~/.bashrc` or
     NODE_HOME=~/tools/node/current
     ...
 
-    export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin
+export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin:$PATH
```

Comment:
are you sure you are previewing it at your commit?
https://github.com/thinkthroughmath/scalr/blob/3e5e7e119b56a3f82066016a809192716e808a04/bin/README.md?l=100#add-to-path